python/action_server/config.py

```diff
@@ -0,0 +1,66 @@
+from pydantic import BaseModel
+import zenoh
+import json
+from typing import Tuple
+
+zenoh.init_logger()
+
+class ZenohValidator(BaseModel):
```

Comment:
ZenohSettings, or ZenohParameters might be better names for this

Reply:
Renamed as ZenohSettings.